$NetBSD: patch-ae,v 1.5 2010/02/21 23:51:26 tron Exp $

Avoid high CPU usage under Mac OS X caused by questionable C++ code.

Please look here for details:
https://bugzilla.gnome.org/show_bug.cgi?id=593240

--- pango/opentype/hb-open-type-private.hh.orig	2009-11-26 00:44:17.000000000 +0000
+++ pango/opentype/hb-open-type-private.hh	2010-02-21 23:41:06.000000000 +0000
@@ -61,7 +61,7 @@
 /* Null objects */
 
 /* Global nul-content Null pool.  Enlarge as necessary. */
-static const char NullPool[32] = "";
+static const void *NullPool[32];
 
 /* Generic template for nul-content sizeof-sized Null objects. */
 template <typename Type>
